tests/cases/fourslash/navigationBarItemsBindingPatternsInConstructor.ts

```diff
@@ -11,4 +11,4 @@
 ////    }
 ////}
 
-verify.navigationBarCount(6); // 2x(class + field + constructor)
+verify.navigationBarCount(9); // global + 2 children + 2x(class + field + constructor)

```

Comment:
this is getting silly. we should change navigationBarCount to take a level, or indent, and count all entries at that level.